### PR TITLE
add flex grid to styl. update responsive grid

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: css/responsive/desktop.styl [hunk 1]
```diff
@@ -0,0 +1,18 @@
+$cols = 12
+$gutter = 32px
+
+.container
+  width: 1150px
+  
+.col-l
+  padding-right: ($gutter / 2)
```
Comment (varffs) [inline css/responsive/desktop.styl]: these values should just be on the `.col` class

File: css/responsive/tablet.styl [hunk 2]
```diff
@@ -0,0 +1,18 @@
+$cols = 12
+$gutter = 28px
+
+.container
+  width: 750px
+  
+.col-m
+  padding-right: ($gutter / 2)
```
Comment (varffs) [inline css/responsive/tablet.styl]: these values should just be on the `.col` class

File: css/site.styl [hunk 3]
```diff
@@ -31,49 +31,86 @@ normalize()
...
+
+// COLUMN SIZES & OFFSETS
+
+.col
+  flex: 1 0 auto
+  box-sizing: border-box
+  max-width: 100%
+  margin-bottom: ($gutter / 2)
+
+.col-s
```
Comment (varffs) [inline css/site.styl:86]: these values should just be on the `.col` class

File: css/site.styl [hunk 4]
```diff
@@ -31,49 +31,86 @@ normalize()
...
+    flex-direction: row-reverse
 
-for col in 1..$cols
-  .col{col}
-    width: ( col * $col ) + ( ( col - 1 ) * $gutter )
+// FLEX POSITIONING
 
-    // FLUID COLS
+.flex-col
+  box-sizing: border-box
```
Comment (varffs) [inline css/site.styl]: these values should just be on the `.col` class too

File: css/site.styl [hunk 5]
```diff
@@ -31,49 +31,86 @@ normalize()
...
+  display: flex
+  flex-direction: column
+  &.reverse
+    flex-direction: column-reverse
 
-.percent-col
-  float: left
+.flex-row
```
Comment (varffs) [inline css/site.styl]: and this can just be `.row` yh?

File: css/site.styl [hunk 6]
```diff
@@ -31,49 +31,86 @@ normalize()
...
+.align-content-end
+  align-content: flex-end
+
+// COLUMN SIZES & OFFSETS
+
+.col
+  flex: 1 0 auto
+  box-sizing: border-box
+  max-width: 100%
+  margin-bottom: ($gutter / 2)
```
Comment (varffs) [inline css/site.styl:84]: i know this can be useful for mobile but perhaps should be left out by default?

Comment (mikeyrayvon) [inline css/site.styl:84]: its for mobile and also so theres margin at the bottom of each row

File: css/responsive/desktop.styl (hunk 1 above)
Comment (mikeyrayvon) [inline css/responsive/desktop.styl]: the problem with having the left and right padding on the `.col` is that it resets the flex, which means we would need to have the repetitive `col-m-6 col-l-6` for example

File: css/site.styl (hunk 6 above)
Comment (varffs) [inline css/site.styl:84]: yh i kno but i perfer to build without auto top or bottom margins but i'll merge and we see how it goes. i like the desktop solution i think

**Pull request merged**